openapi/components/schemas/Websites/WebsiteId.yaml

```
@@ -0,0 +1,8 @@
+type: string
+description: |-
+  Unique website identifier string. A website is where an
+  organization obtains a customer through a subscription. For more
+  information, see
+  [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
```

```suggestion
  Unique website identifier string.
  A website is where an organization obtains a customer through a subscription. For more
  information, see
  [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
```

https://github.com/Rebilly/api-definitions/pull/1077#discussion_r996302156